espresso-support/demo/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,10 @@
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme"
-        tools:ignore="GoogleAppIndexingWarning" />
+        tools:ignore="GoogleAppIndexingWarning">
+
+        <activity android:name=".DemoMainActivity" />
```

Comment:
I'm not sure we do want this anymore - fully qualified seems to be a remnant of old eclipse/early AS. AS is happy with refactoring `.Foo` now.